packages/diagram/pubspec.yaml

```diff
@@ -14,5 +14,9 @@ dev_dependencies:
     sdk: flutter
   image: ^1.0.30
 
+test_dependencies:
+  diagram:
+    path: .
```

Comment:
?

Reply:
Without that, the test couldn't find the diagram package.  I tried `import '../lib/diagram.dart'`, and although that worked, it gave an analyzer warning about not importing things from a 'lib' directory.

Reply:
I'm confused. Where did importing what from where fail?

Reply:
The file `package/diagram/test/diagram_test.dart` was importing with `import '../lib/diagram.dart'`, and it didn't fail, it imported properly, but the analyzer did give a warning.

Adding this test dependency made it so that I could do `import 'packages:diagram/diagram.dart'` in the test instead.

Reply:
Oh, it's because you have the wrong name in the `name` line in the pubspec.yaml.